src/material-experimental/mdc-tabs/_tabs-common.scss

```diff
@@ -17,6 +33,10 @@ $mat-tab-animation-duration: 500ms !default;
     // is an opt-in behavior. Also technically we don't need to combine the two classes, but
     // we need the extra specificity to avoid issues with CSS insertion order.
     flex-grow: 0;
+
+    // MDC seems to include a background color on tabs which only stands
+    // out on dark themes. Disable for now for backwards compatibility.
+    background-color: transparent;
```

Comment:
This lacks enough specificity when users call the theming mixin with additional selectors. It should help if this style was moved to the theming mixin so that it always matches the specificity that the users call it with.